### ISSUE TITLE
Set title and location without pushing state to history

### ISSUE THREAD

**Issue by vmsp** [unpoly/unpoly#188]
There's `up-history` that makes it so the title and location get replaced. The adds an entry to the browser's session history stack. Could we get a similar attribute that uses `history.replaceState` instead?

**Comment by adam12** [comment]
Could you provide a use case? It would be nice to see what you're looking to achieve.

**Comment by vmsp** [comment]
I have a form that uses `up-autosubmit`. The endpoint that's handling this form submission responds with a redirect to the very same form. The title and location of this new page are different, though -- they depend on a form field --, and I want the user to see this change. Using `up-history` does this but it clutters up the user's history.

I rushed the issue's writing and forgot to thank you for this wonderful library. I've tried Hotwire but this definitely takes the cake. The fact that polling is used, instead of WebSockets, is a huge plus.
